tests/config_test.go

```diff
@@ -24,6 +24,10 @@ import (
 	"strings"
 	"time"
 
+	k8sv1 "k8s.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/tests/libvmi"
```

Comment:
does this really deserve its own block? should it not be listed right below `kubevirt.io/kubevirt/tests/` ?

Reply:
To avoid this discussion, we might just use goimports: https://pkg.go.dev/golang.org/x/tools/cmd/goimports

Reply:
(yeah, I admit it is a super tiny nitty nit. Sorry for having raised it)

Reply:
@dankenigsberg Sure, done!